subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationResult.java

```diff
@@ -118,6 +123,11 @@ public String getPropertyNormalizationStrategyName() {
             return propertyNormalizationStrategyIdentifier;
```

Comment:
has to be kept for backward-compat (new Gradle and old build scan plugin)

Reply:
Worth a comment in the code to that effect.